SIL.BuildTasks/UnitTestTasks/TestTask.cs

```diff
@@ -301,7 +301,7 @@ protected void StreamReaderThread_Error()
 					// If looks like an error but includes induce or simulator then log as warning instead of error
 					// Change this if it is still too broad.
 					string[] toerror = { "error", "crash", "fail" };
-					string[] noterror = { "induce", "simulator" };
+					string[] noterror = { "induce", "simulator", "Gdk-CRITICAL **:" };
```

Comment:
I think this needs "Gtk-CRITICAL" as well in the noterror list.  One letter difference from what is there (and which is also needed).